### PR TITLE
Purge carousel from cf.gov

### PR DESCRIPTION
**Pull request by contolini**
Remove front-end assets and carousel model/field because we no longer use a carousel on the homepage.

## Removals

- Everything related to the homepage carousel.

## How to test this PR

1. `./cfgov/manage.py migrate` and `./frontend.sh`
1. The [homepage](http://localhost:8000) should look the same.
1. There should no longer be a carousel tab when [editing the homepage](http://localhost:8000/admin/pages/319/edit/)

### CODE REVIEW

**Comment by chosak** [comment]
:+1: These code changes LGTM, but I think you might need to run [this step](https://cfpb.github.io/consumerfinance.gov/translation/#4-run-the-compilemessages-management-command-to-compile-the-machine-object-file) to fix the failing GHA translations check.

**Comment by contolini** [comment]
> 👍 These code changes LGTM, but I think you might need to run [this step](https://cfpb.github.io/consumerfinance.gov/translation/#4-run-the-compilemessages-management-command-to-compile-the-machine-object-file) to fix the failing GHA translations check.

Ah ty.

**Comment by anselmbradford** [comment]
Translations still failing

File: cfgov/locale/es/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -718,13 +692,6 @@ msgstr ""
 msgid "Test string from Python file."
 msgstr ""
 
-#: v1/tests/management/commands/test_makemessages_data/jinja2/test.html:1
-msgid ""
-"\n"
-"Test string from Jinja template.\n"
-"This is in a multi-line block.\n"
-msgstr ""
-
```
Comment (Scotchester) [inline cfgov/locale/es/LC_MESSAGES/django.po:727]: No idea why this is being removed.

File: cfgov/locale/es/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -1641,6 +1608,14 @@ msgstr "Desempleado o despedido"
...
 msgstr "Jubilado"
 
+#, python-format
+#~ msgid ""
+#~ "Join the conversation. Follow CFPB on %(twitter_link)s and "
+#~ "%(facebook_link)s."
+#~ msgstr ""
+#~ "Únase a la conversación. Síganos en %(twitter_link)s y en "
+#~ "%(facebook_link)s."
+
```
Comment (Scotchester) [inline cfgov/locale/es/LC_MESSAGES/django.po:1618]: The lines in `blog_page.html` that contain this text have not changed in 7 months. Why would they be getting commented out here?

**Review by Scotchester — COMMENTED** [review]
Translation validation is still failing. @cfpb/back-end-admins, we could use your help to figure out why it's behaving this way.

**Comment by anselmbradford** [comment]
I don't know enough about this to know, but do we want to be completely deleting the `.mo` file?

**Comment by anselmbradford** [comment]
Yeah in regard to the `.mo` file, was `compilemessages` run? https://cfpb.github.io/consumerfinance.gov/translation/#4-run-the-compilemessages-management-command-to-compile-the-machine-object-file

**Review by Scotchester — APPROVED** [review]
Looks like the latest effort fixed it!

**Comment by contolini** [comment]
> Yeah in regard to the `.mo` file, was `compilemessages` run? https://cfpb.github.io/consumerfinance.gov/translation/#4-run-the-compilemessages-management-command-to-compile-the-machine-object-file

I'm doing a fresh `makemessages` and `compilemessages` after rebasing. I think part of the issue is that some translations were changed upstream and the portable object file wasn't rebuilt.

**Comment by Scotchester** [comment]
> I'm doing a fresh `makemessages` and `compilemessages` after rebasing. I think part of the issue is that some translations were changed upstream and the portable object file wasn't rebuilt.

@contolini Yeah, it looked correct and was working for me before you updated from main, but now the validate-translations check is failing again, so I think you're probably right.

**Comment by contolini** [comment]
story

**Pull request merged**